rest_framework/filters.py

```diff
@@ -24,15 +24,19 @@ def search_smart_split(search_terms):
     """generator that first splits string by spaces, leaving quoted phrases together,
```

Comment:
Came from [your haiku](https://krakentech.slack.com/archives/C02CXGKKHL0/p1711105770012189)
Just to add :panda_face: comment
Docstring needs fix

(“generator” -> Create list)

Reply:
I'll put up a PR for it!